src/app/digital/actions/ports/CoderPortChangeAction.ts

```diff
@@ -53,7 +53,10 @@ export class CoderPortChangeAction implements Action {
      * @param val refers to the target number chosen by the user.
      */
     protected changeSize(val: number): void {
-        this.obj.setSize(V(DEFAULT_SIZE, DEFAULT_SIZE/2 * Math.pow(2, val)));
+        let mul: number = 1;
+        if(val >= 5) mul = 1.2;
+        if(val == 8) mul = 1.4;
```

Comment:
Should probably just have this be a continuous function instead of having it change every 5,8 steps
So like just `DEFAULT_SIZE * (1 + (val - 1)/20)` or something that works